storage/tianmu/exporter/export2file.cpp

```diff
@@ -52,7 +50,7 @@ int select_tianmu_export::prepare(List<Item> &list, Query_expression *u) {
   fixed_row_size = (!field_term_length && !exchange->field.enclosed->length() && !blob_flag);
 
   prepared = true;
-  return 0;
+  return true;
```

Comment:
Is the function always returns `true`?

Reply:
it should always returns false,i wil correct it. 
1 Base class of select_tianmu_export  is Query_result. In Query_result:: prepare, it said that "**returns false if success, true if error**", so false means success;
2 There is only one return-branch in the original tianmu code,
maybe someday, we can add some code to return true if we know a new senario.